en/onboarding.ftl

```diff
@@ -61,3 +61,40 @@ onboarding-premium-reply-description-2 = Need to reply to emails sent to a mask?
 multi-part-onboarding-premium-reply-description = Reply to emails without giving away your real address
 
 multi-part-onboarding-step-counter = Step { $step } of { $max }.
+
+# New free user onboarding strings (November 8th, 2023)
```

Comment:
This comment only applies to the next string.

Since it's more of a developer note, you should leave an empty line after, so it becomes a stand-alone comment (not displayed in Pontoon to localizers).